### PR TITLE
Frontend 1893 - Add Aside Component to Dev-Center

### PR DESCRIPTION
**Pull request by langlk**
Resolves [FRONTEND-1893](https://algorithmia.atlassian.net/browse/FRONTEND-1893)

Relies on style changes in https://github.com/algorithmiaio/synapse/pull/186

After going down a couple _very_ deep, fruitless rabbit holes, I think this is the solution that is best balance between ease of use and minimal fragility. Creating an aside in a markdown page file looks like this:
```
{% include aside-start.html %}
# Left-hand content aside is associated with
{% include aside-middle.html %}
# Right-hand aside content
{% include aside-end.html %}
```
I've added some examples in [this branch](https://github.com/algorithmiaio/dev-center/compare/FRONTEND-1893...FRONTEND-1893-example?expand=1)

### CODE REVIEW

File: css/devcenter.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 @import "../synapse/code";
 @import "../synapse/variables";
 @import "../synapse/typography-modifiers";
+@import "../synapse/article";
```
Comment (langlk) [inline css/devcenter.scss]: This is a temporary addition, will remove once the corresponding synapse PR is merged and built

**Comment by bnlb** [comment]
http://localhost:4000/developers/tutorials/recipes/

This page looks a bit awkward. Doesn't appear related to the aside changes, but wanted to mention it so we don't miss it before merging to `develop`.

**Comment by langlk** [comment]
> LL a usable pattern, TY!
> 
> Presumably I can have several aside sections in the same doc?

Yep! See [here](https://github.com/algorithmiaio/dev-center/compare/FRONTEND-1893...FRONTEND-1893-example?expand=1) for an example!

**Comment by bnlb** [comment]
Thanks for adding the example PR, makes this much easier to grok. 💯

**Comment by langlk** [comment]
> This looks really good. Can you add some documentation on how to use these? I guarantee I'll forget and be hunting for the description in this PR sometime in the future.

Good call!

**Pull request merged**